packages/core-lib/__tests__/query/__tests__/index.test.ts

```diff
@@ -0,0 +1,303 @@
+            const queryHelper = new QueryHelper();
+            const apiFilters = queryHelper.setFilters([{
+                k: 'time',
+                v: '2021-05-01',
+                o: '=t',
+            }]).apiQuery.filter;
+            expect(apiFilters).toEqual([
+                { k: 'time', v: '2021-05-01T00:00:00.000Z', o: 'datetime_gte' },
+                { k: 'time', v: '2021-05-02T00:00:00.000Z', o: 'datetime_lt' },
+            ]);
+        });
+        test('date value and ">t" operator => "datetime_gte"', () => {
+            const queryHelper = new QueryHelper();
+            const apiFilters = queryHelper.setFilters([{
+                k: 'time',
+                v: '2021-05-01',
+                o: '>t',
+            }]).apiQuery.filter;
+            expect(apiFilters).toEqual([
+                { k: 'time', v: '2021-05-01T00:00:00.000Z', o: 'datetime_gte' },
+            ]);
+        });
+        test('date value and ">=t" operator => "datetime_gte"', () => {
+            const queryHelper = new QueryHelper();
+            const apiFilters = queryHelper.setFilters([{
+                k: 'time',
+                v: '2021-05-01',
+                o: '>=t',
+            }]).apiQuery.filter;
+            expect(apiFilters).toEqual([
+                { k: 'time', v: '2021-05-01T00:00:00.000Z', o: 'datetime_gte' },
+            ]);
+        });
+        test('date value and "<t" operator => "datetime_lte"', () => {
+            const queryHelper = new QueryHelper();
+            const apiFilters = queryHelper.setFilters([{
+                k: 'time',
+                v: '2021-05-01',
+                o: '<t',
+            }]).apiQuery.filter;
+            expect(apiFilters).toEqual([
+                { k: 'time', v: '2021-05-01T00:00:00.000Z', o: 'datetime_lte' },
+            ]);
+        });
+        test('date value and "<=t" operator => "datetime_lte"', () => {
+            const queryHelper = new QueryHelper();
+            const apiFilters = queryHelper.setFilters([{
+                k: 'time',
+                v: '2021-05-01',
+                o: '<=t',
+            }]).apiQuery.filter;
+            expect(apiFilters).toEqual([
+                { k: 'time', v: '2021-05-01T00:00:00.000Z', o: 'datetime_lte' },
+            ]);
+        });
+        test('datetime value and "=t", ">t" operator => "datetime_gt"', () => {
+            const queryHelper = new QueryHelper();
+            const apiFilters = queryHelper.setFilters([{
+                k: 'time',
+                v: '2021-05-01 12:00:00',
+                o: '=t',
+            }, {
+                k: 'time',
+                v: '2021-05-01 12:00:00',
+                o: '>t',
+            }]).apiQuery.filter;
+            expect(apiFilters).toEqual([
+                { k: 'time', v: '2021-05-01T12:00:00.000Z', o: 'datetime_gt' },
+                { k: 'time', v: '2021-05-01T12:00:00.000Z', o: 'datetime_gt' },
```

Comment:
Question: Why are the above two operator values the same?
```
    '>t': 'datetime_gt',
    '=t': 'datetime_gt',
```

Reply:
That's good question!
The default operator for detatime is 'gt'.
And there is no correct api operator for '=t'.
<img width="591" alt="image" src="https://github.com/cloudforet-io/console/assets/26986739/03725981-b495-4c78-b2b6-282c559c0834">